### DISCUSSION TITLE
Error while trying to update nuclei on macOS Monterey

### DISCUSSION THREAD

**Discussion by who0sy** [projectdiscovery/nuclei#1627] [Q&A] [answered]
Trying to update nuclei from 2.5.4 to 2.5.7 on macos Monterey but gives me an build error.

**Comment by forgedhallpass** [comment, marked as answer]
Hello @who0sy,

Please use the issue template when creating issues, and include all the requested information like error logs, screenshots etc.

Please also note, that we only provide support for the latest version of nuclei, which is `v2.6.0`.
